### DISCUSSION TITLE
是不是不好继续用vercel部署了？

### DISCUSSION THREAD

**Discussion by wyfang** [jason5ng32/MyIP#226] [Q&A] [answered]
好像是达到免费的限额了？
`Error: No more than 12 Serverless Functions can be added to a Deployment on the Hobby plan. Create a team (Pro plan) to deploy more. Learn More: https://vercel.link/function-count-limit`

**Comment by feko2018** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/bcd3b0fb-8c9f-4ee5-82bc-f3cefda87087)

要把api目录改成其他的，apis等，
再把下面这几个文件里面的/api 改成/apis   即可。
